nixos/modules/installer/cd-dvd/iso-image.nix

```diff
@@ -658,13 +691,17 @@ in
       '';
     };
 
-    isoImage.graphicalGrub = mkOption {
+    isoImage.forceTextMode = mkOption {
       default = false;
       type = types.bool;
       example = true;
       description = lib.mdDoc ''
-        Whether to use textmode or graphical grub.
-        false means we use textmode grub.
+        Whether to use text mode or graphical grub.
+        A value of `false` means graphical mode is not tried to be used.
```

Comment:
shouldn't it be the opposite, forceTextMode set to *true* disables graphical mode?

Reply:
Right, that was from the initial rewrite where I kept the `graphicalGrub` option name; good catch.